prac_08/silver_service_taxi_test.py

```diff
@@ -0,0 +1,20 @@
+
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+
+def main():
+    name = input("Car Name: ")
```

Comment:
We don't want to test with user input. Imagine running your tests 100 times per day...